src/components/ChangePlan/Checkout/ContactInformation/ContactInformation.js

```diff
@@ -0,0 +1,239 @@
+import React, { useState } from 'react';
+import { useIntl } from 'react-intl';
+import useTimeout from '../../../../hooks/useTimeout';
+import {
+  FieldGroup,
+  FieldItem,
+  FormWithCaptcha,
+  InputFieldItem,
+  PhoneFieldItem,
+  SelectFieldItem,
+  SubmitButton,
+} from '../../../form-helpers/form-helpers';
+
+export const ContactInformation = ({
+  contactInformation,
+  countries,
+  activeStep,
+  setActiveStep,
+  handleSaveAndContinue,
+}) => {
+  const intl = useIntl();
+  const _ = (id, values) => intl.formatMessage({ id: id }, values);
+  const stepName = 'contact-information';
+
+  const fieldNames = {
+    firstname: 'firstname',
+    lastname: 'lastname',
+    address: 'address',
+    city: 'city',
+    province: 'province',
+    country: 'country',
+    zipCode: 'zipCode',
+    phone: 'phone',
+    company: 'company',
+    industry: 'industry',
+  };
+
+  const getFormInitialValues = () => {
+    let initialValues = Object.keys(fieldNames).reduce(
+      (accumulator, currentValue) => ({
+        ...accumulator,
+        [currentValue]: '',
+      }),
+      {},
+    );
+
+    if (contactInformation) {
+      initialValues = { ...contactInformation };
+    }
+
+    return initialValues;
+  };
+
+  const [formSubmitted, setFormSubmitted] = useState(false);
+  const createTimeout = useTimeout();
+
+  const submitContactInformationForm = async (values, { setSubmitting }) => {
+    setFormSubmitted(false);
+    try {
+      const result = await handleSaveAndContinue(values);
+      if (result.success) {
+        setFormSubmitted(true);
+      }
+    } finally {
+      createTimeout(() => setFormSubmitted(false), 3000);
+    }
+  };
+
+  const edit = () => {
+    setActiveStep(stepName);
+  };
+
+  /* TODO: These styles will be removed with the correct classes. Currently Gus is working with the layout. */
+  const containerStyle = {
+    paddingLeft: '20px',
+    paddingTop: '20px',
+    paddingRight: '20px',
+    border: '2px solid #eaeaea',
+    borderRadius: '3px',
+  };
+  const containerMessageSucces = { padding: '0px' };
+  const flextStartSyle = { justifyContent: 'flex-start' };
+  const displaFlexAndjustifyContentSpaceBetweenStyle = {
+    display: 'flex',
+    justifyContent: 'space-between',
+  };
+
+  return (
+    <div className="dp-wrapper-form-plans contact-information" style={containerStyle}>
+      <div style={displaFlexAndjustifyContentSpaceBetweenStyle}>
+        <h3>{_('checkoutProcessForm.contact_information_title')}</h3>
+        {activeStep !== stepName ? (
+          <a href="#" onClick={edit}>
+            Edit
```

Comment:
It should be in the language file, right?

Reply:
This is a temporary beacuse it will be replaced with an icon and for it reason I didn't add it in the language files

Reply:
Do not worry, add it anyway